sys/sys/sysproto.h

```diff
@@ -29,15 +29,6 @@ struct thread;
 #if BYTE_ORDER == LITTLE_ENDIAN
 #define	PADL_(t)	0
 #define	PADR_(t)	PAD_(t)
-#elif defined(_MIPS_SZCAP) && _MIPS_SZCAP == 256
-/*
- * For non-capability arguments, the syscall argument is stored in the
- * cursor field in the second word.
- */
```

Comment:
This was always a bug. If we ever do 256 again we should properly respect the endianness and put the cursor last in memory.